### PR TITLE
Update hscs launch plans with technical topics, day in the life, courses and books

### PR DESCRIPTION
**Pull request by thierrychau**

### CODE REVIEW

File: cohorts/we-2024-7-1-sdf-bridge/launch-plans.md [hunk 1]
```diff
@@ -230,26 +230,34 @@
...
-
-- [Eloquent JavaScript](./javascript.md#eloquent-javascript) 
+- [Introduction to JavaScript from Scrimba](./javascript.md#introduction-to-javascript)
 <!-- TODO: azure -->
 - 
 
 ### Books:
-<!-- TODO -->
+- [Eloquent JavaScript](./javascript.md#eloquent-javascript) 
+- [Patterns of Enterprise Application Architecture](https://martinfowler.com/books/eaa.html)
```
Comment (heratyian) [inline cohorts/we-2024-7-1-sdf-bridge/launch-plans.md:256]: I swapped this out with the spring book. Can you update?

File: cohorts/we-2024-7-1-sdf-bridge/launch-plans.md [hunk 2]
```diff
@@ -230,26 +230,34 @@
...
+### Day in the life:
+- [A day in the life of a Java Software Engineer by Learn with Phil](https://www.youtube.com/watch?v=aCP4tDSpjGQ)
+- [A day in the life of a Spring Developer by ProCoder](https://www.youtube.com/watch?v=eBWm1l1AZF8)
+
 
 ### Courses:
 
 <!-- mostly strong java is most important -->
 - [Hyperskill: Introduction to Java](./java.md#hyperskill-introduction-to-java)
 - [Hyperskill: Java Backend Developer (Spring Boot)](./java.md#hyperskill-java-backend-developer-spring-boot)
```
Comment (heratyian) [inline cohorts/we-2024-7-1-sdf-bridge/launch-plans.md:249]: I added learncs.online as another good free java resource. Good to add especially with hyperskill free tier limits.

**Review by heratyian — CHANGES_REQUESTED** [review]

File: cohorts/we-2024-7-1-sdf-bridge/launch-plans.md (hunk 2 above)
Comment (thierrychau) [reply cohorts/we-2024-7-1-sdf-bridge/launch-plans.md:249]: Updated

File: cohorts/we-2024-7-1-sdf-bridge/launch-plans.md (hunk 1 above)
Comment (thierrychau) [reply cohorts/we-2024-7-1-sdf-bridge/launch-plans.md]: Updated

**Review by heratyian — APPROVED** [review]
Looks great. Go ahead and update canvas.

**Pull request merged**